oneview_redfish_toolkit/api/redfish_json_validator.py

```diff
@@ -22,7 +22,7 @@
 from oneview_redfish_toolkit.api.errors \
```

Comment:
Copyright of this file should be updated.

Reply:
Done.